pkg/handlers/session.go

```diff
@@ -47,21 +47,26 @@ func parseClusterAuthorization(authHeader string) (authorization, error) {
 	}, nil
 }
 
-func requireValidSession(kotsStore store.Store, w http.ResponseWriter, r *http.Request) (*sessiontypes.Session, error) {
+func requireValidSession(kotsStore store.Store, w http.ResponseWriter, r *http.Request) (sess *sessiontypes.Session, err error) {
 	if r.Method == "OPTIONS" {
 		return nil, nil
 	}
 
 	auth := r.Header.Get("authorization")
```

Comment:
same question regarding the `Authorization` header

Reply:
We need to keep the check for the authorization header in this case because this function is also used to validate sessions for requests done via the CLI. Those will still use a `Kots` token set in the authorization header.